dpanonymize/__init__.py

```diff
@@ -89,7 +99,7 @@ def get_file_objects_from_module(root_dir: Union[Path, str],
         - module: name of the module to remove PII from, str.
         - BIDS: True if the PHOENIX is in BIDS structure, bool.
     '''
-    module_dirs = list(root_dir.glob(f'*/{module}')) if BIDS else \
+    module_dirs = list(root_dir.glob(f'*/*/*/{module}')) if BIDS else \
```

Comment:
Do you remember where we talked about one `*` versus two `*`?

Reply:
No, can you remind me about this if you remember it please?